app/react/Library/components/PermissionsFilter.tsx

```diff
@@ -1,66 +1,145 @@
-import React from 'react';
-import SelectFilter from 'app/Library/components/SelectFilter';
+import React, { useMemo } from 'react';
 import FormGroup from 'app/DocumentForm/components/FormGroup';
+import { MultiSelect } from 'app/Forms';
 import { Aggregations } from 'shared/types/aggregations';
 import { NeedAuthorization } from 'app/Auth';
 import { Translate } from 'app/I18N';
-import { Icon } from 'app/UI';
+import { Control } from 'react-redux-form';
+import { connect } from 'react-redux';
+import { UserSchema } from 'shared/types/userType';
+import { IImmutable } from 'shared/types/Immutable';
+import Immutable from 'immutable';
+
+type PermissionLevel = 'read' | 'write';
+
+interface PermissionsValue {
```

Comment:
There's the same thing in the AssigneeFilter. Would it not make sense to have it in one place, for consistency?

Reply:
On it

Reply:
And today I've learnt that 'filtrable' is a valid spelling of the word. :)